qt/applications/workbench/workbench/plotting/plotscriptgenerator/figure.py

```diff
@@ -0,0 +1,28 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+
+from __future__ import (absolute_import, unicode_literals)
+
+from workbench.plotting.plotscriptgenerator.utils import convert_args_to_string
+
+BASE_CREATE_FIG_COMMAND = "plt.figure({})"
+
+
+def get_figure_command_kwargs(fig):
+    kwargs = {
+        'figsize': (fig.get_figwidth(), fig.get_figheight()),
+        'dpi': fig.dpi,
+        'num': fig.get_label()
+    }
+    return kwargs
+
+
+def generate_figure_command(fig):
```

Comment:
Was it considered that we could use [`plt.subplots`](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.subplots.html) to create the figure and axes together? Our [documentation](https://docs.mantidproject.org/nightly/plotting/index.html#introduction-to-matplotlib-in-mantid) uses it so it would be good to keep things inline with what we tell users in documentation and what we use ourselves. It would also be more compact when coming to create subplots later on...